pkg/asset/installconfig/gcp/gcp.go

```diff
@@ -38,16 +38,48 @@ func selectProject(ctx context.Context) (string, error) {
 	}
 	defaultProject := ssn.Credentials.ProjectID
 
+	client := &Client{
+		ssn: ssn,
+	}
+
+	var listOfCombo []string
+
+	listOfNames, listOfIds, err := client.GetListOfProjects(ctx)
+	if err != nil {
+		return "", err
+	}
+	index := 0
+	for i, v := range listOfIds {
+		if v == defaultProject {
+			index = i
+		}
+		listOfCombo = append(listOfCombo, listOfNames[i]+" ("+v+")")
+	}
+
+	defaultValue := listOfNames[index] + " (" + defaultProject + ")"
+
 	var selectedProject string
 	err = survey.Ask([]*survey.Question{
 		{
-			Prompt: &survey.Input{
-				Message: "Project ID",
-				Help:    "The project id where the cluster will be provisioned. The default is taken from the provided service account.",
-				Default: defaultProject,
+			Prompt: &survey.Select{
+				Message: "Project",
+				Help:    "The GCP project to be used for installation.",
+				Default: fmt.Sprintf("%s", defaultValue),
+				Options: listOfCombo,
 			},
+			Validate: survey.ComposeValidators(survey.Required, func(ans interface{}) error {
```

Comment:
I think `survey` already limits choices to `Options` so I don't think this extra validator is needed.